src/node/mod.rs

```diff
@@ -142,15 +139,7 @@ impl Node {
         self.item.set_fscore(fscore);
     }
 
-    pub fn compute_item_fscore(
-        &mut self,
-        previous_state: &Goban,
-        to_play: &BitBoard,
-        depth: usize
-    ) -> Fscore {
-        self.item.compute_fscore(previous_state, to_play, depth)
-    }
-
+    #[allow(unused)]
```

Comment:
why?

Reply:
Because this method (and the one below as well) is not currently used, and it produces a warning at compilation but I don't want to remove this method (yet) because I'm not totally sure (yet) that it would not be needed in a near future.
However, I can comment those methods instead if this is what you would like.